package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "action-cli-runscript",
+  "version": "1.0.0",
+  "description": "A GitHub Action that runs an Office 365 CLI script",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Would you please be able to add `build` task with the command you've used to generate the dist/index.js file?

I assume it is something like `tsc -p tsconfig.json`, but it would be nice to have it here documented and ready to be run again in a case of changes.

Reply:
Thank you @VelinGeorgiev . I have added that. The build task is `ncc build src\\main.ts`. One of the dev dependencies is `@zeit/ncc`. If we use `tsc` then we will have to check in `node_modules` as well. Hence using `ncc` as suggested in the [GitHub Actions documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action#commit-and-push-your-action-to-github).